readme.md

```diff
@@ -110,7 +110,7 @@ Default: `false`
 
 ```js
 queryString.parse('foo[]=1&foo[]=2&foo[]=3', {parseNumbers: true});
-//=> foo: [1, 2, 3]
+//=> {'foo[]': [1, 2, 3]}
```

Comment:
I think we need a simpler example here. Maybe just `queryString.parse('foo=1', {parseNumbers: true});`